prog/download_boot_image.rb

```diff
@@ -72,7 +76,10 @@ def blob_storage_client
         version: version,
         sha256sum: sha256_sum
       }.to_json
-      sshable.cmd("common/bin/daemonizer 'host/bin/download-boot-image #{params_json.shellescape}' #{q_daemon_name}", stdin: Config.ubicloud_images_blob_storage_certs)
+      sshable.cmd(
+        "common/bin/daemonizer 'host/bin/download-boot-image #{params_json.shellescape}' #{q_daemon_name}",
+        stdin: download_from_blob_storage? ? Config.ubicloud_images_blob_storage_certs : nil
```

Comment:
What do you think about moving `certs` property to `params_json` and pass whole json via stdin?